src/scripts/components/Play.js

```diff
@@ -160,7 +161,7 @@ const PlayInfoButtons = styled.div`
   margin: 15px 0 15px;
 `
 
-const ButtonIcon = styled(Button)`
+const ButtonIcon = styled(TextButton)`
   display: flex;
```

Comment:
This is part of Icon button discussion, because we need that to put the icon and text side by side and align(-items) them on center.
Anyway, I'll `extend`the TextButton for now

Reply:
I created a `IconButtonStyle` for now
```
<TextButton iconButton>
 <SVGIcon icon="icon-player-play"  />
 <TranslatedText />
</TextButton>
```
https://github.com/Paratii-Video/paratii-portal/blob/0da75f91ffe239b5311e64c65e29d2cf8e342fac/src/scripts/components/foundations/Button.js#L89-L92